Main.java

```diff
@@ -3,74 +3,121 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.*;
 import java.util.StringTokenizer;
 
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    static final String FILE_TYPE_TXT = ".txt";
+    static final String FILE_TYPE_JSON = ".json";
+    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
+    static final String SINGLE_SPACE = "";
+    static String searchResult;
 
     public static void main(String[] args) {
 
+        String textFilePath;
+        String searchKeyword;
+        String errorMessage;
         if (args.length != 2) {
-            System.out.println("File path and Search word cannot be empty");
+            errorMessage = "Provide valid filepath and search keyword";
+            System.out.println(errorMessage);
+            Database.databaseOperation(null, null, searchResult = "Failure", errorMessage, 0);
             return;
         }
-        String textFilePath = args[0];
-        String searchKeyword = args[1];
+        textFilePath = args[0];
+        searchKeyword = args[1];
         File file = new File(textFilePath);

```

Comment:
add a an extra condition for null ro empty for both filePath and searchword

Reply:
Sure. Will do that.